etcd/config.go

```diff
@@ -0,0 +1,48 @@
+package etcd
+
+import (
+	"time"
+)
+
+// Config defines the config for storage.
+type Config struct {
+	// Endpoints is a list of URLs.
+	Endpoints []string
+	// DialTimeout is the timeout for failing to establish a connection.
+	DialTimeout time.Duration
+	// Username is a username for authentication.
+	Username string
+	// Password is a password for authentication.
+	Password string
```

Comment:
It may be worth adding some of the other Config options here, like the autosync, keepalive, and message size options. Nice to have would be adding the "TLS" config option.

Src: https://pkg.go.dev/go.etcd.io/etcd/client/v3#Config

Reply:
TLS Config option added https://github.com/gofiber/storage/pull/878/commits/d95d8e93b9869420b989cb3d46368eec51fe878d .